_contributors/baird_alan.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Alan Baird
+twitter: bluegrasscoder
+location: Austin, TX
+website: http://twitter.com/bluegrasscoder
```

Comment:
I don't have anything here yet but you might as well use this: http://abaird.github.io/